packages/back-end/src/types/Integration.ts

```diff
@@ -188,6 +191,25 @@ export type ExperimentMetricQueryResponse = {
   covariate_sum_squares?: number;
   main_covariate_sum_product?: number;
 }[];
+export type QueryResponse = {
+  // eslint-disable-next-line
+  rows: any[];
+  statistics?: QueryStatistics;
+};
```

Comment:
Better to use generics here.

```ts
// Add a named generic 'Row' here
export type QueryResponse<Row> = {
  rows: Row[];
  statistics?: QueryStatistics;
}

// Now, the other definitions become much simpler
export type MetricValueQueryResponse = QueryResponse<MetricValueQueryResponseRow>;
```

Reply:
I knew there had to be a way to do this, thank you so much!